app/forms/gpo_verify_form.rb

```diff
@@ -41,6 +41,7 @@ def submit(is_enhanced_ipp)
       success: result,
       errors: errors,
       extra: {
+        profile_age_in_seconds: pending_profile&.profile_age_in_seconds,
```

Comment:
Not your problem here, but there sure is a lot of `pending_profile&` in this method

Reply:
There have been some weird edge cases in the past with profile being nil for really bizarre reasons. 😬